src/main/java/org/elasticsearch/index/query/GeoBoundingBoxQueryParser.java

```diff
@@ -183,7 +183,7 @@ public Filter parse(QueryParseContext parseContext) throws IOException, QueryPar
         }
```

Comment:
It will happen anyway as we replace filters with queries internally, but I can try to find all the bad guys